plugins/catalog-backend/api-report.md

```diff
@@ -172,6 +173,7 @@ export type CatalogEnvironment = {
   config: Config;
   reader: UrlReader;
   permissions: PermissionEvaluator | PermissionAuthorizer;
+  eventBroker?: EventBroker;
```

Comment:
similar to `PluginEnvironment`:
I wonder whether we should just call this `events` like the service 🤔